build/passes-linux.yml

```diff
@@ -0,0 +1,14 @@
+
+steps:
+- script: |
```

Comment:
It would be good to give this a friendly name so that it is labeled in the build details page. Right now it just shows up with the generic name "CmdLine" (see https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=30149&view=logs&j=743bc1cd-037d-5150-fc69-2953b556a708&t=ac832828-8db5-5d4a-de7c-12645fac0708&l=1)

Reply:
```suggestion
- name: Linux Passes Build & Tests
   script: |
```